exercicios/para-casa/entregas/eliane-almeida/classBook.js

```diff
@@ -0,0 +1,54 @@
+import { Author } from './classAuthor.js'
+
+export class Book extends Author {
+    #nameBook;
+    #isbn;
+    #price;
+    #qty;
+
+    constructor(nameAuthor, nameBook, isbn, price, qty) {
+        super(nameAuthor)
+
+        this.#nameBook = nameBook;
+        this.#isbn = isbn;
+        this.price = price;
```

Comment:
@ElianeA no this.price não faria mais sentido estar como privado '#'. Tendo em vista que ela está sendo declarada acima dessa forma ?